tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
Ideally do we agree that we should remove that class instead of making more tests extend it (because unit tests don't need any environment)?

However I definitely understand that you took the simplest path right now because the diff is big enough already ;)